c/tskit/tables.c

```diff
@@ -6005,7 +6029,7 @@ tsk_table_collection_build_index(tsk_table_collection_t *self, tsk_flags_t TSK_U
 {
     int ret = TSK_ERR_GENERIC;
     size_t j;
-    double *time = self->nodes.time;
+    double *times = self->nodes.time;
```

Comment:
This has nothing to do with the changes in question --- please keep changes focused on a specific thing and avoid 'drive-by' changes like this. The singular usage is deliberate here and consistent elsewhere.

Reply:
I've reverted back to `time` for an array of multiple times, as you had, and used `node_time` for the single instance variable. I'm not sure if this is what you want, given the comment on line 5605? Perhaps it should be the other way around?

Reply:
This is fine --- as I say, there' no right answer here.